contactaccessrights/CRM/Contactaccessrights/Test/Fabricator/Rights.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+/**
+ * Fabricates rights for tests.
+ */
+abstract class CRM_Contactaccessrights_Test_Fabricator_Rights {
+
+  /**
+   * Fabricates Rights, given specific values for contact_id, entity_id and 
+   * entity_type.
+   * 
+   * @param array $params
+   *   Associative array of parameters to be used to store rights infrmation in 
+   *   database.
+   * @return array
+   *   Holds values stored in database for created acess right
+   * @throws Exception
+   * @throws CiviCRM_API3_Exception
+   */
+  public static function fabricate($params) {
+    try {
+      $result = civicrm_api3('Rights', 'create', [
+        'sequential' => 1,
+        'contact_id' => $params['contact_id'],
+        'entity_id' => $params['entity_id'],
+        'entity_type' => $params['entity_type']
+      ]);
+    } catch (CiviCRM_API3_Exception $e) {
+      throw $e;

```

Comment:
This just doesn't make sense. If you're just throwing the catched exception you don't need the try catch block at all